Libraries/LibGfx/PainterSkia.cpp

```diff
@@ -60,6 +64,198 @@ static SkPathFillType to_skia_path_fill_type(Gfx::WindingRule winding_rule)
+                    skia_color_filter = SkColorFilters::Matrix(matrix, SkColorFilters::Clamp::kNo);
+                    break;
+                }
+                case Gfx::ColorFilter::Type::Contrast: {
+                    float intercept = -(0.5f * amount) + 0.5f;
+                    float matrix[20] = {
+                        amount, 0, 0, 0, intercept,
+                        0, amount, 0, 0, intercept,
+                        0, 0, amount, 0, intercept,
+                        0, 0, 0, 1, 0
+                    };
+                    skia_color_filter = SkColorFilters::Matrix(matrix, SkColorFilters::Clamp::kNo);
+                    break;
+                }
+                case Gfx::ColorFilter::Type::Invert: {
+                    float matrix[20] = {
+                        1 - 2 * amount, 0, 0, 0, amount,
+                        0, 1 - 2 * amount, 0, 0, amount,
+                        0, 0, 1 - 2 * amount, 0, amount,
+                        0, 0, 0, 1, 0
+                    };
+                    skia_color_filter = SkColorFilters::Matrix(matrix, SkColorFilters::Clamp::kYes);
+                    break;
+                }
+                case Gfx::ColorFilter::Type::Opacity: {
+                    float matrix[20] = {
+                        1, 0, 0, 0, 0,
+                        0, 1, 0, 0, 0,
+                        0, 0, 1, 0, 0,
+                        0, 0, 0, amount, 0
+                    };
+                    skia_color_filter = SkColorFilters::Matrix(matrix, SkColorFilters::Clamp::kYes);
+                    break;
+                }
+                case Gfx::ColorFilter::Type::Sepia: {
+                    float matrix[20] = {
+                        0.393f + 0.607f * (1 - amount), 0.769f - 0.769f * (1 - amount), 0.189f - 0.189f * (1 - amount), 0, 0,
+                        0.349f - 0.349f * (1 - amount), 0.686f + 0.314f * (1 - amount), 0.168f - 0.168f * (1 - amount), 0, 0,
+                        0.272f - 0.272f * (1 - amount), 0.534f - 0.534f * (1 - amount), 0.131f + 0.869f * (1 - amount), 0, 0,
+                        0, 0, 0, 1, 0
+                    };
+                    skia_color_filter = SkColorFilters::Matrix(matrix, SkColorFilters::Clamp::kYes);
+                    break;
+                }
+                case Gfx::ColorFilter::Type::Saturate: {
+                    float matrix[20] = {
+                        0.213f + 0.787f * amount, 0.715f - 0.715f * amount, 0.072f - 0.072f * amount, 0, 0,
+                        0.213f - 0.213f * amount, 0.715f + 0.285f * amount, 0.072f - 0.072f * amount, 0, 0,
+                        0.213f - 0.213f * amount, 0.715f - 0.715f * amount, 0.072f + 0.928f * amount, 0, 0,
+                        0, 0, 0, 1, 0
+                    };
+                    skia_color_filter = SkColorFilters::Matrix(matrix, SkColorFilters::Clamp::kNo);
+                    break;
+                }
+                default:
+                    VERIFY_NOT_REACHED();
```

Comment:
This enum class cannot just get a random non-enum value from unverified source, and compile-time check seems to me to be a more reasonable solution here. I suggest removing this VERIFY_NOT_REACHED, so if there's a new enum case not present here, it fails to compile instead of a runtime failure